src/alchemtest/amber/access.py

```diff
@@ -106,3 +111,30 @@ def load_invalidfiles():
 
     return Bunch(data=data,
                  DESCR=fdescr)
+
+
+def load_testfiles():
+    """Load incomplete-wrongly formatted files to be used to test the AMBER parsers.
+
+    Returns
+    -------
+    data : Bunch
+        Dictionary-like object, the interesting attributes are:
+
+        - 'data' : the data files
+        - 'DESCR': the full description of all the files
+
+    """
+
+    module_path = dirname(__file__)
+
+    data = {}
+    for f in listdir(join(module_path, 'testfiles')):
+        if f.endswith('bz2'):
+            data[f[:-12]] = [join(module_path, 'testfiles', f)]
```

Comment:
The filename extraction is fragile. Use functions in os.path (basename, splitext, etc) to generate the pure filename without the extension. Or use pathlib.

Reply:
I resolved using pathlib. The method I implemented requires the file to end with `.bz2`, as it was before, but now removes anything in ('.tar', '.bz2', '.out'), allowing to use `.` in the filename, if needed, and allowing to have files with different complex extensions.